components/raftstore/src/store/fsm/apply.rs

```diff
@@ -3941,12 +3941,8 @@ where
 
         self.delegate.term = apply.term;
         if let Some(meta) = apply.bucket_meta.clone() {
-            let buckets = self
-                .delegate
-                .buckets
-                .get_or_insert_with(BucketStat::default);
-            buckets.stats = new_bucket_stats(&meta);
-            buckets.meta = meta;
+            let new = BucketStat::from_meta(meta);
+            self.delegate.buckets.replace(new);
```

Comment:
This will refresh the creation time of the buckets.

Reply:
Good catch, the `create_time` is useless, I will remove it.

Reply:
why?

Reply:
yes, I have added new function to modify meta and stats.